apps/admin.py

```diff
@@ -4,12 +4,38 @@
 
 
 class AppsAdmin(admin.ModelAdmin):
-    list_display = ("name", "user_can_create", "slug", "revision")
+    list_display = (
```

Comment:
It seems like this change undoes the formatting of black. Have you run the black formatter on the fetched code?